docs/router/framework/solid/guide/path-params.md

```
@@ -1,15 +1,769 @@
+    </div>
+  )
+}
+```
+
+### Type Safety with Optional Parameters
+
+TypeScript provides full type safety for optional parameters:
+
+```tsx
+function PostsComponent() {
+	// TypeScript knows category might be undefined
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)